core/admin/migrations/versions/8f9ea78776f4_.py

```diff
@@ -0,0 +1,21 @@
+"""empty message
+
+Revision ID: 8f9ea78776f4
+Revises: 3b7eee912b41
+Create Date: 2022-03-11 13:53:08.996055
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '8f9ea78776f4'
+down_revision = '3b7eee912b41'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('user', sa.Column('spam_mark_as_read', sa.Boolean(), nullable=False))
```

Comment:
This yields an error:
```
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) Cannot add a NOT NULL column with default value NULL
[SQL: ALTER TABLE user ADD COLUMN spam_mark_as_read BOOLEAN NOT NULL]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
```
Please add the default: `server_default=sa.sql.expression.false()`